packages/forms-web-app/src/routes/lpa-dashboard/statement.js

```diff
@@ -13,6 +13,9 @@ const {
 const getJourneyResponse = require('../../dynamic-forms/middleware/get-journey-response-for-lpa-statement');
 const setDefaultSection = require('../../dynamic-forms/middleware/set-default-section');
 const redirectToUnansweredQuestion = require('../../dynamic-forms/middleware/redirect-to-unanswered-question');
+const {
+	skipIfNoAdditionalDocuments
+} = require('..//../dynamic-forms/middleware/redirect-middleware-conditions');
```

Comment:
```suggestion
} = require('../../dynamic-forms/middleware/redirect-middleware-conditions');
```
unless `//` is a pathing syntax I don't know (it might well be)

Reply:
Ah sorry, must've double tapped the key

Reply:
Just pushed a correction!